src/source.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+
+export default class Source extends React.Component {
+
+  render() {
```

Comment:
I'm a little bit new to React (coming from a background of mainly AngularJS 1.x and node) so I'm still learning some of the best practices (this is a great example).

I am not, although I do murder code sometimes 👿